### PR TITLE
Task/APPS-2665 — intake form updates

### PR DESCRIPTION
**Pull request by giuliacampana**
- move the intake date to the top of the form, to its own section
- make sure form date and status date are the same

### CODE REVIEW

File: src/containers/intake/schemas/IntakeSchemas.js [hunk 1]
```diff
@@ -236,6 +243,12 @@ const dataSchema = {
 
 const uiSchema = {
   [getPageSectionKey(1, 1)]: {
+    classNames: 'column-span-12 grid-container',
+    [getEntityAddressKey(0, FORM, DATETIME_ADMINISTERED)]: {
+      classNames: 'column-span-4'
+    },
+  },
```
Comment (anzioka) [inline src/containers/intake/schemas/IntakeSchemas.js:250]: Widget? 😀

**Review by anzioka — APPROVED** [review]

File: src/containers/intake/schemas/IntakeSchemas.js [hunk 2]
```diff
@@ -51,6 +51,19 @@ const dataSchema = {
...
   properties: {
     [getPageSectionKey(1, 1)]: {
+      type: 'object',
+      title: '',
+      properties: {
+        [getEntityAddressKey(0, FORM, DATETIME_ADMINISTERED)]: {
+          type: 'string',
+          title: 'Intake Date',
+          format: 'date',
+          default: DateTime.fromISO(currentDateTime).toISODate()
```
Comment (UnsungHero97) [inline src/containers/intake/schemas/IntakeSchemas.js:61]: Why not just `DateTime.local().toISODate()`?

**Review by UnsungHero97 — APPROVED** [review]

File: src/containers/intake/schemas/IntakeSchemas.js (hunk 1 above)
Comment (giuliacampana) [reply src/containers/intake/schemas/IntakeSchemas.js:250]: ooh you only have to put `format: 'date'` in the schema for the property to be a DateWidget!

File: src/containers/intake/schemas/IntakeSchemas.js (hunk 2 above)
Comment (giuliacampana) [reply src/containers/intake/schemas/IntakeSchemas.js:61]: hmm when i first made this form, i wanted the intake date and status date to be exactly the same so i saved `DateTime.local().toISO()` to a variable to be used for each, but it doesn't really make sense... could've just saved `DateTime.local()` to a variable, but also now, i handle saving the exact same datetime in the `onSubmit`.

File: src/containers/intake/schemas/IntakeSchemas.js (hunk 1 above)
Comment (anzioka) [reply src/containers/intake/schemas/IntakeSchemas.js:247]: good to know! Thanks :)

**Review by anzioka — APPROVED** [review]

**Review by UnsungHero97 — APPROVED** [review]

**Pull request merged**